src/components/shared/SrOnly.js

```diff
@@ -0,0 +1,18 @@
+import { Text } from 'grommet'
+import styled from 'styled-components'
+
+const Wrapper = styled(Text)`
```

Comment:
You can rename this to be more descriptive like `HiddenText`

Reply:
This is a commonly used name for text specifically intended for screen readers(screen-reader-only), so in this case,`HiddenText` might be too ambiguous. Let's keep this as is or we can also rename it `SROnly`. Let me know, thank you!

Reply:
Removed the wrapper(commented in below), thank you! 
https://github.com/AlexsLemonade/refinebio-web/pull/31#issuecomment-1329595464